src/ui-lib/sass/00-settings/_animations.scss

```diff
@@ -2,4 +2,5 @@
    Animation Variables
 */
 $grav-tr-speed: 250ms;
+$grav-tr-speed-slow-2: 600ms;
```

Comment:
you can call it `$grav-tr-speed-slower` ? I know it's a shitty way to name it, but it's better than `-2` IMO